evap/staff/tests/test_views.py

```diff
@@ -2383,6 +2383,30 @@ def test_finishing_review_updates_results(self):
         self.assertEqual(len(results.questionnaire_results[0].question_results[1].answers), 1)
 
 
+class TestEvaluationTextAnswersSkip(WebTestStaffMode):
+    csrf_checks = False
+
+    def test_skip(self):
+        manager = make_manager()
+        student = baker.make(UserProfile, email="student@institution.example.com")
+        evaluation = baker.make(
+            Evaluation,
+            participants=[student],
+            voters=[student],
+            state="in_evaluation",
+            can_publish_text_results=True,
+        )
+        top_general_questionnaire = baker.make(Questionnaire, type=Questionnaire.Type.TOP)
+        baker.make(Question, questionnaire=top_general_questionnaire, type=Question.TEXT)
+        evaluation.general_contribution.questionnaires.set([top_general_questionnaire])
+        baker.make(TextAnswer, contribution=evaluation.general_contribution)
+
+        skip_url = "/staff/textanswers/skip"
+        self.app.post(skip_url, user=manager, status=200, params={
+            'evaluation_id': evaluation.id
+        })
```

Comment:
You should test some behaviour in your test, here it would be to check that the skipped evaluation is saved in the session. This works on my machine: (why `self.client.session` is not working, but `repsonse.client.session` does – I don’t know)
```suggestion
        response = self.app.post(skip_url, user=manager, status=200, params={
            'evaluation_id': evaluation.id,
        })
        self.assertEqual(response.client.session['review-skipped'], {evaluation.id})
```